include/sys/zio.h

```diff
@@ -163,63 +163,62 @@ typedef enum zio_suspend_reason {
 	ZIO_SUSPEND_MMP,
 } zio_suspend_reason_t;
 
-enum zio_flag {
+typedef uint64_t zio_flag_t;
```

Comment:
While it may be obvious to some, let's add a brief comment here explaining an `enum` was not used because it's limited to 32-bits.

Reply:
Done.